### PR TITLE
Swedish translations. Issues #248, #249, #216, #196 and #127

### PR DESCRIPTION
**Pull request by matsduf**
First commit just arranges sv.po in the same way as en.po and adds missing tags. The following commits updates the translations.

### CODE REVIEW

**Comment by matsduf** [comment]
@mattias-p ! Can you review?

**Comment by matsduf** [comment]
I have built the equivalent .mo file from the sv.po by the make file, and I have tested in the CLI.

**Comment by mattias-p** [comment]
In the quotes commit, most double quotes and parentheses are replaced with single quotes. In a few occasions combined parentheses and single quotes are used. Is there some guiding principle for when different quotations are to be used which or is decided arbitrarily by the translator?

File: share/sv.po [hunk 1]
```diff
@@ -129,81 +203,145 @@ msgstr "[ASN:ANNOUNCE_BY] {address};{asn}"
...
 
 #: CONSISTENCY:SOA_RNAME
 msgid  "Saw SOA rname {rname} on following nameserver set : {servers}."
-msgstr "Såg SOA RNAME {rname} på följande namnservrar : {servers}."
+msgstr "'SOA RNAME' '{rname}' hittades på följande namnservrar: {servers}."
 
 #: CONSISTENCY:SOA_SERIAL
 msgid  "Saw SOA serial number {serial} on following nameserver set : {servers}."
-msgstr "Såg SOA-serienummer {serial} på följande namnservrar : {servers}."
+msgstr "SOA-serienummeret {serial} hittades på följande namnservrar: {servers}."
```
Comment (mattias-p) [inline share/sv.po]: "serienummeret" -> "serienumret"

File: share/sv.po [hunk 2]
```diff
@@ -129,81 +203,145 @@ msgstr "[ASN:ANNOUNCE_BY] {address};{asn}"
...
+
+#: CONSISTENCY:SOA_SERIAL_VARIATION
+msgid  "Difference between the smaller serial ({serial_min}) and the bigger one ({serial_max}) is greater than the maximum allowed ({max_variation})."
+msgstr "Spannet mellan högsta och lägsta SOA-serienummeret är större än {max_variation}."
```
Comment (mattias-p) [inline share/sv.po]: "serienummeret" -> "serienumret"

File: share/sv.po [hunk 3]
```diff
@@ -217,101 +355,117 @@ msgstr "Minsta möjliga giltiga hänvisningspaket är på 512 oktetter eller min
...
+msgid  "The DNSKEY with tag {keytag} uses deprecated algorithm number {algorithm}/({description})."
+msgstr "DNSKEY-posten med 'tag' {keytag} använder algoritm nummmr {algorithm}/({description}), som inte längre ska användas."
+
+#: DNSSEC:ALGORITHM_OK
+msgid  "The DNSKEY with tag {keytag} uses algorithm number {algorithm}/({description}), which is OK."
+msgstr "DNSKEY-posten med 'tag' {keytag} använder algoritm nummer {algorithm}/({description}), vilken är OK."
+
+#: DNSSEC:ALGORITHM_PRIVATE
+msgid  "The DNSKEY with tag {keytag} uses private algorithm number {algorithm}/({description})."
+msgstr "DNSKEY-posten med 'tag' {keytag} använder en algoritm med det privata algoritmnummeret {algorithm}/({description})."
```
Comment (mattias-p) [inline share/sv.po]: "algoritmnummeret" -> "algoritmnumret"

File: share/sv.po [hunk 4]
```diff
@@ -217,101 +355,117 @@ msgstr "Minsta möjliga giltiga hänvisningspaket är på 512 oktetter eller min
...
+
+#: DNSSEC:ALGORITHM_RESERVED
+msgid  "The DNSKEY with tag {keytag} uses reserved algorithm number {algorithm}/({description})."
+msgstr "DNSKEY-posten med tag {keytag} använder en algoritm med det reserverade algoritm nummeret {algorithm}/({description})."
```
Comment (mattias-p) [inline share/sv.po]: "algoritm nummeret" -> "algoritmnumret"

File: share/sv.po [hunk 5]
```diff
@@ -217,101 +355,117 @@ msgstr "Minsta möjliga giltiga hänvisningspaket är på 512 oktetter eller min
...
+
+#: DNSSEC:ALGORITHM_UNASSIGNED
+msgid  "The DNSKEY with tag {keytag} uses unassigned algorithm number {algorithm}/({description})."
+msgstr "DNSKEY-posten med tag {keytag} använder en algoritm med det otilldelade algoritmnummeret {algorithm}/({description})."
```
Comment (mattias-p) [inline share/sv.po]: "algoritmnummeret" -> "algoritmnumret"

File: share/sv.po [hunk 6]
```diff
@@ -353,97 +507,125 @@ msgstr "Antalet NSEC3-iterationer är {count}, vilket är högt."
...
+msgstr "RRSIG med 'keytag' {tag} täckande posttyp(er) {types} löper ut datum {date}."
+
+#: DNSSEC:RRSIG_EXPIRED
+msgid  "RRSIG with keytag {tag} and covering type(s) {types} has already expired (expiration is: {expiration})."
+msgstr "RRSIG-posten med {tag} och täckande posttyperna {types} har redan löpt ut (utgångstiden är: {expiration})."
 
 #: DNSSEC:SOA_NOT_SIGNED
 msgid  "No RRSIG correctly signed the SOA RRset."
-msgstr "Ingen RRSIG signerade korrekt SOA-mängden."
+msgstr "Det finns ingen RRSIG som korrekt signerade SOA-posten."
```
Comment (mattias-p) [inline share/sv.po]: "signerade" -> "signerat"

File: share/sv.po [hunk 7]
```diff
@@ -353,97 +507,125 @@ msgstr "Antalet NSEC3-iterationer är {count}, vilket är högt."
...
+msgstr "Delegeringen från moderzonen till dotterzonens namnservrar är inte korrekt signerad ({reason})."
 
 #: DNSSEC:DELEGATION_SIGNED
 msgid  "Delegation from parent to child is properly signed."
-msgstr "Delegeringen från föräldra- till barn-zon är korrekt signerad."
+msgstr "Delegeringen från moderzonen till dotterzonens namnservrar är korrekt signerad."
+
+#: EXAMPLE:EXAMPLE_TAG
+msgid  "This is an example tag."
+msgstr "Detta är en exempeltagg."
```
Comment (mattias-p) [inline share/sv.po]: "exempeltagg" -> "exempeletikett"?

File: share/sv.po [hunk 8]
```diff
@@ -455,135 +637,213 @@ msgstr "Namnservern {ns}/{address} tillåter zonöverföringar med AXFR."
...
+#: NAMESERVER:IPV4_DISABLED
+#: NAMESERVER:IPV6_DISABLED
 #: NAMESERVER:IS_A_RECURSOR
 msgid  "Nameserver {ns}/{address} is a recursor."
-msgstr "Namnservern {ns}/{address} är en rekurserande resolver."
+msgstr "Namnservern {ns}/{address} är en rekursiv resolver."
+
+#: NAMESERVER:NO_RECURSOR
+msgid  "None of the following nameservers is a recursor : {names}."
+msgstr "Ingen av följande namnservrar är rekursiv resolver: {names}."
```
Comment (mattias-p) [inline share/sv.po]: "är rekursiv resolver" -> "är en rekursiv resolver"

File: share/sv.po [hunk 9]
```diff
@@ -455,135 +637,213 @@ msgstr "Namnservern {ns}/{address} tillåter zonöverföringar med AXFR."
...
+
+#: NAMESERVER:RECURSIVITY_UNDEF
+msgid  "Can not determine nameservers recursivity."
+msgstr "Det går inte att fastställa om namnserver är rekursiv resolver eller inte."
```
Comment (mattias-p) [inline share/sv.po]: "om namnserver är rekursiv resolver" -> one of:
* "om namnservern är en rekursiv resolver"
* "om en namnserver är en rekursiv resolver"
* "om en namnservrarna är rekursiva resolvrar"

File: share/sv.po [hunk 10]
```diff
@@ -455,135 +637,213 @@ msgstr "Namnservern {ns}/{address} tillåter zonöverföringar med AXFR."
...
+msgstr "Det finns otillåtna tecken i namnservernamnet '{name}'."
 
 #: SYNTAX:NAMESERVER_NUMERIC_TLD
 msgid  "Nameserver ({name}) within a 'numeric only' TLD ({tld})."
-msgstr "Namnservern ({name}) finns i en helnumerisk TLD ({tld})."
+msgstr "Namnservern ('{name}') finns under en helnumerisk TLD ('{tld}')."
+
+#: SYNTAX:NAMESERVER_SYNTAX_OK
+msgid  "Nameserver ({name}) syntax is valid."
+msgstr "Namnservernamnet ser OK ut."
```
Comment (mattias-p) [inline share/sv.po]: "Namnservernamnet ser OK ut." -> "Namnservernamnet är giltigt."

File: share/sv.po [hunk 11]
```diff
@@ -645,21 +913,9 @@ msgstr "Begäran om att köra den okända metoden {method} i modulen {module}."
...
-msgstr "Lade till en artificiell delegering för domänen {domain} hos namnserver {ns}."
-
-#: SYSTEM:FAKE_DELEGATION_TO_SELF
-msgid  "Name server {ns} not adding fake delegation for domain {domain} to itself."
-msgstr "Namnservern {ns} lade inte till en artificiell delegering för domänen {domain}."
-
 #: SYSTEM:PACKET_BIG
 msgid  "Packet size ({size}) exceeds common maximum size of {maxsize} bytes (try with \"{command}\")."
-msgstr "Packet size ({size}) exceeds common maximum size of {maxsize} bytes (try with \"{command}\")."
+msgstr "Paketstorleken ({size}) Byte överskrider den normalt maximala storleken om {maxsize} Byte. Försök med kommandot '{command}'."
```
Comment (mattias-p) [inline share/sv.po]: "Byte" -> "byte" (twice)

**Comment by matsduf** [comment]
> In the quotes commit, most double quotes and parentheses are replaced with single quotes. 

In the Swedish translations no double quotes are used, only single.

> In a few occasions combined parentheses and single quotes are used. Is there some guiding principle for when different quotations are to be used which or is decided arbitrarily by the translator?

I do not know of any guiding principles, but I do think the messages will need more work in the future to make them more uniform.

**Pull request merged**